.circleci/config.yml

```diff
@@ -86,8 +61,8 @@ jobs:
       - run:
           name: Run ruff
           command: |
+            ruff format --check
             ruff check
-            # ruff format
```

Comment:
The ruff format docs recommend `format` ___after___ `check`.

Reply:
Thanks! Fixed!